### PR TITLE
Add documentation for stop method

### PR DESCRIPTION
**Pull request by tkyi**
Resolves https://github.com/screwdriver-cd/executor-k8s/issues/31

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ The parameters required are:
 | config.token | String | JWT to act on behalf of the build |
 | callback | Function | Callback for when task has been created |
 
-The `_start` function will start a job in kubernetes with labels for easy lookup. These labels are:
+The `_start` function will start a job in Kubernetes with labels for easy lookup. These labels are:
```
Comment (d2lam) [inline README.md]: `start`

File: README.md [hunk 2]
```diff
@@ -46,6 +46,24 @@ The callback is called with:
...
+
+The parameters required are:
+
+| Parameter        | Type  |  Description |
+| :-------------   | :---- | :-------------|
+| config        | Object | Configuration Object |
+| config.buildId | String | The unique ID for a build |
+| callback | Function | Callback for when task has been created |
+
+The `_stop` function will stop a job in Kubernetes using a label:
```
Comment (d2lam) [inline README.md]: `stop`

File: README.md [hunk 3]
```diff
@@ -46,6 +46,24 @@ The callback is called with:
...
+### Stop
+The `_stop` method takes advantage of the input validation defined in the [executor-base-class].
+
+The parameters required are:
+
+| Parameter        | Type  |  Description |
+| :-------------   | :---- | :-------------|
+| config        | Object | Configuration Object |
+| config.buildId | String | The unique ID for a build |
+| callback | Function | Callback for when task has been created |
```
Comment (d2lam) [inline README.md]: `when job has been stopped`

File: README.md [hunk 4]
```diff
@@ -46,6 +46,24 @@ The callback is called with:
...
+
+The `_stop` function will stop a job in Kubernetes using a label:
+* sdbuild: config.buildId
+
+The callback is called with:
+* An error `callback(err)` when an error occurs starting the job
```
Comment (d2lam) [inline README.md]: stoping

File: README.md [hunk 5]
```diff
@@ -46,6 +46,24 @@ The callback is called with:
...
+* null `callback(null)` when a job is correctly started
```
Comment (d2lam) [inline README.md]: stopped

File: README.md [hunk 6]
```diff
@@ -18,8 +18,8 @@ The class provides a couple options that are configurable in the instantiation o
 | Parameter        | Type  |  Description |
 | :-------------   | :---- | :-------------|
 | config        | Object | Configuration Object |
-| config.token | String | The JWT token used for authenticating to the kubernetes cluster |
-| config.host | String | The hostname for the kubernetes cluster i.e. `kubernetes` if running inside kubernetes |
+| config.token | String | The JWT token used for authenticating to the Kubernetes cluster |
+| config.host | String | The hostname for the Kubernetes cluster i.e. `Kubernetes` if running inside Kubernetes |
 
 ### Start
 The `_start` method takes advantage of the input validation defined in the [executor-base-class].
```
Comment (FenrirUnbound) [inline README.md]: `start`

File: README.md [hunk 7]
```diff
@@ -46,6 +46,24 @@ The callback is called with:
 * An error `callback(err)` when an error occurs starting the job
 * null `callback(null)` when a job is correctly started
 
+### Stop
+The `_stop` method takes advantage of the input validation defined in the [executor-base-class].
```
Comment (FenrirUnbound) [inline README.md]: `stop`

**Comment by d2lam** [comment]
👍

**Pull request merged**